UI/reconciliation/report.html

```diff
@@ -220,14 +261,16 @@
         name = "id_$i",
         value = row.id
     } ?>
-        <tr class="record">
-            <td><?lsmb INCLUDE input element_data = {
+        <tr class="record" id=<?lsmb "recon-line-$row.id" ?>>
+            <td><?lsmb IF !approved and !submitted;
+            INCLUDE input element_data = {
             type = "checkbox"
             name = "cleared_$row.id"
             value = row.id
             checked = row.cleared
-            }
-            ?></td>
+            "data-dojo-type"  = "lsmb/ReconciliationLine"
```

Comment:
On the invoice, the `InvoiceLine` matches the `<tbody><tr>...</tr></tbody>` tags in the table representing invoice lines. Isn't it confusing to have these named `...Line` when they are just a checkbox?